var/spack/repos/builtin/packages/mvapich2/package.py

```diff
@@ -91,7 +91,7 @@ class Mvapich2(AutotoolsPackage):
     variant(
         'file_systems',
         description='List of the ROMIO file systems to activate',
-        values=auto_or_any_combination_of('lustre', 'gpfs', 'nfs', 'ufs'),
+        values=auto_or_any_combination_of('ime', 'lustre', 'gpfs', 'nfs', 'ufs'),
```

Comment:
```suggestion
        values=auto_or_any_combination_of(
            'ime', 'lustre', 'gpfs', 'nfs', 'ufs'
        ),
```
Maybe that'll make flake8 happy